doc/source/testing.rst

```diff
@@ -143,18 +166,18 @@ the next gold standard iteration.  You can see the current iteration by looking
 in the ``.travis.yml`` file at the ``TRIDENT_GOLD`` entry--enumerate this and
```

Comment:
Not your change, but I think "enumerate" should be "increment".

Reply:
Agreed.